conda/resolve.py

```diff
@@ -179,31 +168,10 @@ def installed(self):
         # type: () -> Set[Dist]
         installed = set()
         for dist, info in iteritems(self.index):
-            if 'link' in info and not dist.with_features_depends:
+            if 'link' in info:
```

Comment:
No need for this, because `supplement_index_from_prefix` is only going to tag the records that were not populated with a `with_features_depends` Dist.